schema.yaml

```diff
@@ -249,6 +249,121 @@ types:
     type: object
     required:
       - name
+  cloud-toolkit-aws:kubernetes:ApplicationAddonArgs:
+    properties:
+      k8sProvider:
+        description: ""
+        $ref: /kubernetes/v3.21.2/schema.json#/provider
+      name:
+        description: ""
+        type: string
+      namespace:
+        description: ""
+        type: string
+      createNamespace:
+        description: ""
+        type: boolean
+    type: object
+    required:
+      - k8sProvider
+      - name
+      - namespace
+  cloud-toolkit-aws:kubernetes:IrsaApplicationAddonArgs:
+    properties:
+      identityProvidersArn:
+        description: ""
+        type: array
+        items:
+          description: ""
+          type: string
+      issuerUrl:
+        description: ""
+        type: string
+      serviceAccountName:
+        description: ""
+        type: string
+    type: object
+    required:
+      - identityProvidersArn
+      - issuerUrl
+      - serviceAccountName
+  cloud-toolkit-aws:kubernetes:ArgoCDArgs:
+    properties:
+      hostname:
+        description: ""
+        type: string
+    type: object
+    required: []
+  cloud-toolkit-aws:kubernetes:CertManagerArgs:
+    properties:
+      zoneArns:
+        description: ""
+        type: array
+        items:
+          description: ""
+          type: string
+    type: object
+    required:
+      - zoneArns
+  cloud-toolkit-aws:kubernetes:ClusterAddonsArgs:
+    properties:
+      k8sProvider:
+        description: ""
+        $ref: /kubernetes/v3.21.2/schema.json#/provider
+      identityProvidersArn:
+        description: ""
+        type: array
+        items:
+          description: ""
```

Comment:
I would avoid empty descriptions in the schema. I suggest searching for the properties and functions that are missing these descriptions and briefly define them at least.